src/api/spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe SearchController, vcr: true do
+  render_views
+
+  shared_examples 'block anonymous request' do
+    it { expect(response).to have_http_status(:redirect) }
```

Comment:
I don't think this makes much sense. `require_login` is a "default" `before_filter` from `ApplicationController`.

Reply:
done